view/graph/GraphArea.kt

```diff
@@ -197,11 +191,12 @@ class GraphArea(transactionState: TransactionState) {
 
     @Composable
     private fun VertexLayer() {
```

Comment:
### Vertex rendering

- If vertex is entirely outside viewport, don't draw it
- If viewport scale < 0.2 (20%), OR if (no. of vertices in viewport > 100 AND physics alpha > 0.25), don't draw vertex labels. This ensures smooth performance when zoomed out and during the initial explosion.